Cargo.lock

```diff
@@ -1063,6 +1067,19 @@ dependencies = [
  "wasm-bindgen",
 ]
 
+[[package]]
+name = "const-hex"
```

Comment:
This should replace hex crate, right? I think @onur-ozkan suggested we can do without both. I would suggest not adding new deps if the benefit is not much.

Reply:
reverted